front/app/lib/api/heist/index.ts

```diff
@@ -0,0 +1,36 @@
+import { gql, type GraphQLClient } from 'graphql-request';
+
+import type { Query } from '~api/types';
+
+export const getDayHeists = async (client: GraphQLClient) => {
+  const tomorrow = new Date();
+  tomorrow.setDate(tomorrow.getDate() + 1);
+  tomorrow.setHours(0, 0, 0, 0);
+
+  return client.request<Query>(
```

Comment:
```suggestion
  return client.request<Query>(
```

Regarder les autres query pour utiliser un `Pick` et le bon payload